libs/go/libp2p_node/aea/api.go

```diff
@@ -346,7 +346,9 @@ func (aea *AeaApi) Connect() error {
 	aea.closing = false
 	//TOFIX(LR) trade-offs between bufferd vs unbuffered channel
 	aea.out_queue = make(chan *Envelope, 10)
-	go aea.listen_for_envelopes()
+	aea.send_queue = make(chan *Envelope, 10)
```

Comment:
Buffers seem low. When the buffer is exceeded it will block on 144 and potentially cause a deadlock ..

Reply:
fixed